src/track/track.h

```diff
@@ -375,6 +379,8 @@ class Track : public QObject {
 
     mixxx::TrackRecord m_record;
 
+    double dFirstPhraseBegin;
```

Comment:
Members are prefixed with `m_`. Why is this member directly placed in the Track object instead of encapsulating it in the `Beats` class or one of its subclasses? I don't think that it belongs here.